openapi/SwarmCommon.yaml

```diff
@@ -655,6 +655,10 @@ components:
           $ref: "#/components/schemas/BigInt"
         gasLimit:
           type: integer
+        gasTipCap:
```

Comment:
did you increment the version on this?

Reply:
nope, but if i rem last time, we decided to not increment nums i guess because there were too many changes happening in api.

Reply:
please update

Reply:
nope, but if i rem last time, we decided to not increment nums i guess because there were too many changes happening in api.
if its required i can do it but then there are multiple apis where we didnt update these, specifically during api refactoring epic because so many changes were happening, should i?